tsl/test/expected/cagg_errors.out

```diff
@@ -607,3 +607,29 @@ ERROR:  invalid continuous aggregate view
 -- No FROM clause in CAGG definition
 CREATE MATERIALIZED VIEW cagg1 with (timescaledb.continuous, timescaledb.materialized_only=false) AS SELECT 1 GROUP BY 1 WITH NO DATA;
 ERROR:  invalid continuous aggregate query
+--
+-- Create cagg with UUIDv7 time column is currently unsupported
+--
+CREATE TABLE uuid_part (id UUID NOT NULL, device INT, temp FLOAT);
+SELECT create_hypertable('uuid_part', 'id');
+    create_hypertable    
+-------------------------
+ (19,public,uuid_part,t)
+(1 row)
+
+INSERT INTO uuid_part VALUES
+       ('0194214e-cd00-7000-a9a7-63f1416dab45', 2, 2.0),
+       ('01942117-de80-7000-8121-f12b2b69dd96', 1, 1.0),
+       ('0194263e-3a80-7000-8f40-82c987b1bc1f', 3, 3.0),
+       ('01942675-2900-7000-8db1-a98694b18785', 4, 4.0),
+       ('01942bd2-7380-7000-9bc4-5f97443907b8', 5, 5.0),
+       ('01942d52-f900-7000-866e-07d6404d53c1', 6, 6.0);
+CREATE MATERIALIZED VIEW uuid_part_daily WITH (timescaledb.continuous, timescaledb.materialized_only=false)
+AS
+SELECT
+  time_bucket('1 day', _timescaledb_functions.timestamptz_from_uuid_v7(id)),
+  SUM(temp)
+FROM uuid_part
+GROUP BY 1
+ORDER BY 2 DESC;
+ERROR:  time bucket function must reference the primary hypertable dimension column
```

Comment:
This error is a little misleading, or at least confusing. The time bucket function _does_ reference the primary hypertable dimension column, it is just using a function between. Might be a good idea to add a "directly" and maybe have the details mention this in better terms.